std_msgs/String task
---
zeabus_vision/vision_qualifying_gate data


